webui/main.py

```diff
@@ -83,11 +84,15 @@ def get_incomplete_task():
 @app.post("/task")
 def add_completion_task(completion_task: TaskCreator):
     uuid = db_add_completion_task(completion_task.prompt, completion_task.mode)
+    result = db_get_completion_tasks_by_uuid(uuid)
+    while result is None or result["completion_result"] is None:
+        result = db_get_completion_tasks_by_uuid(uuid)
     return {
-        "task_uuid": uuid,
+        "summary": result["completion_result"]
     }
```

Comment:
Since you're immidiately returning `result["completion_result"]` just after adding the task, you'll return it's result before it's even been processed, resulting either in an error, or an empty undefined value.
Previous approach, the one returning `uuid` would force the frontend to track this particular task, or at least acknowledge it, but now no useful data is being passed.
We're also introducing new keywords which are not present in the schema, the schema predicts the usage of `completion_result`, but you're returning a `summary`.
Please fix.